File: run_batch.js
const { execSync } = require('child_process');

try {
  const output = execSync('cmd /c C:\\Users\\ANURON\\smartHire-BE-FE-v2\\create_and_verify_dirs.bat', {
    encoding: 'utf-8',
    stdio: 'pipe',
    windowsHide: false
  });
  console.log(output);
} catch (error) {
  console.error('Error:', error.message);
  if (error.stdout) console.log('Output:', error.stdout);
  if (error.stderr) console.log('Error Output:', error.stderr);
  process.exit(1);
}

File: run_batch.py
import subprocess
import sys

try:
    result = subprocess.run(
        r'cmd /c C:\Users\ANURON\smartHire-BE-FE-v2\create_and_verify_dirs.bat',
        capture_output=True,
        text=True,
        shell=True
    )
    print(result.stdout)
    if result.stderr:
        print("STDERR:", result.stderr)
    sys.exit(result.returncode)
except Exception as e:
    print(f"Error: {e}")
    sys.exit(1)
